cloud-config/master.yaml.tmpl

```diff
@@ -2171,8 +2179,8 @@ coreos:
       -v /etc/kubernetes/ssl/:/etc/kubernetes/ssl/ \
       quay.io/giantswarm/certctl:b07d0913d5cb369a6b605394bdd4be4633451be9 \
       issue \
-      --vault-addr=https://${VAULT_DOMAIN_NAME} \
-      --vault-token=${G8S_VAULT_TOKEN} \
+      --vault-addr=$${VAULT_ADDR} \
+      --vault-token=$${VAULT_TOKEN} \
```

Comment:
`$$` -> `$` ?

Reply:
Noop, it's intentionally, See here https://www.terraform.io/docs/providers/template/d/file.html#inline-templates

`$$` used to escape `$`

Reply:
Ok, my bad
lgtm then